src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -0,0 +1,192 @@
+
+    errors += SparkMaxUtils.check(beamBreakSensorOne.enableLimitSwitch(false));
+    errors += SparkMaxUtils.check(beamBreakSensorTwo.enableLimitSwitch(false));
+    errors += SparkMaxUtils.check(beamBreakSensorThree.enableLimitSwitch(false));
+    errors += SparkMaxUtils.check(beamBreakSensorFour.enableLimitSwitch(false));
+
+    return errors == 0;
+  }
+
+  /** Stops the conveyor rollers. */
+  private void stopRollers() {
+    frontMotor.set(0.0);
+    backMotor.set(0.0);
+  }
+
+  /** Send note to the shooter. */
+  public static Command shoot(Conveyor conveyor) {
+    return new SequentialCommandGroup(
+        new InstantCommand(
+            () -> conveyor.frontMotor.set(ConveyorCal.PREPARE_TO_SHOOT_FRONT_SPEED), conveyor),
+        new InstantCommand(() -> conveyor.backMotor.set(ConveyorCal.PREPARE_TO_SHOOT_BACK_SPEED)),
+        new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPositions.PARTIAL_NOTE),
+        new WaitCommand(ConveyorCal.NOTE_EXIT_TIME_SHOOTER_SECONDS),
+        Conveyor.stop(conveyor),
+        new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPositions.NO_NOTE));
+  }
+
+  /** Score note into the trap or the amp */
+  public static Command scoreTrapOrAmp(Conveyor conveyor) {
+    return new SequentialCommandGroup(
+        new InstantCommand(
+            () -> conveyor.frontMotor.set(ConveyorCal.SCORE_AMP_TRAP_FRONT_SPEED), conveyor),
+        new InstantCommand(() -> conveyor.backMotor.set(ConveyorCal.SCORE_AMP_TRAP_BACK_SPEED)),
+        new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPositions.PARTIAL_NOTE),
+        new WaitCommand(ConveyorCal.NOTE_EXIT_TIME_TRAP_AMP_SECONDS),
+        Conveyor.stop(conveyor),
+        new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPositions.NO_NOTE));
+  }
+
+  /** Get a note from the intake. */
+  public static Command receive(Conveyor conveyor) {
+    return new SequentialCommandGroup(
+        new InstantCommand(() -> conveyor.backMotorEncoder.setPosition(0.0), conveyor),
+        new InstantCommand(() -> conveyor.frontMotor.set(ConveyorCal.FRONT_RECEIVE_SPEED)),
+        new InstantCommand(() -> conveyor.backMotor.set(0.0)),
+        new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPositions.PARTIAL_NOTE),
+        new WaitUntilCommand(
+            () ->
+                Math.abs(conveyor.backMotorEncoder.getPosition())
+                    > ConveyorCal.NOTE_POSITION_THRESHOLD_IN),
+        new InstantCommand(() -> conveyor.backMotorEncoder.setPosition(0.0)),
+        new InstantCommand(() -> conveyor.frontMotorEncoder.setPosition(0.0)),
+        new InstantCommand(() -> conveyor.frontMotor.set(ConveyorCal.BACK_OFF_POWER)),
+        new InstantCommand(() -> conveyor.backMotor.set(ConveyorCal.BACK_OFF_POWER)),
+        new WaitUntilCommand(
+            () -> conveyor.frontMotorEncoder.getPosition() < ConveyorCal.BACK_OFF_IN),
```

Comment:
i would change to BACK_OFF_INCHES rather than BACK_OFF_IN to be more clear

Reply:
Yep fixed